src/sql/azdata.proposed.d.ts

```diff
@@ -823,11 +823,61 @@ declare module 'azdata' {
 		parentTypeName?: string;
 	}
 
+	/**
+	 * Represents a selected range in the result grid.
+	 */
+	export interface SelectionRange {
+		fromRow: number;
+		toRow: number;
+		fromCell: number;
```

Comment:
Not sure what happened to my comment, but this should really be `fromColumn` and `toColumn` IMO